sponsorship_compassion/wizards/generate_gift_wizard.py

```diff
@@ -53,9 +53,12 @@ def generate_invoice(self):
                             logger.error(
                                 'The birthdate of the child is missing!')
                             continue
-                        invoice_date = self.compute_date_birthday_invoice(
-                            contract.child_id.birthdate,
-                            self.invoice_date)
+                        if self.env.context.get('lsv'):

```

Comment:
You have this test but the value `lsv` is never put in the context! Please add it in the place where the automatic gifts are generated.